forge-gui/res/cardsfolder/upcoming/helpful_hunter.txt

```diff
@@ -0,0 +1,7 @@
+Name:Helpful Hunter
+ManaCost:1 W
+Types:Creature Cat
+PT:1/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ When this creature enters, draw a card.
+SVar:TrigDraw:DB$ Draw | Defined$ You | NumCards$ 1
```

Comment:
DB$ Draw
nothing else needed ;)

Reply:
Fair enough. While I was at it, I went back to the sources of the code for these and did the same cleanup there.